mlir/lib/Dialect/LLVMIR/IR/LLVMTypes.cpp

```diff
@@ -959,26 +862,12 @@ Type mlir::LLVM::getVectorType(Type elementType,
 }
 
 Type mlir::LLVM::getFixedVectorType(Type elementType, unsigned numElements) {
```

Comment:
I guess that we can burn out these API's in a subsequent step? Maybe we should mark them as deprecated now?

Reply:
I'm going to do another pass over this file (after this has been merged) and try to remove unnecessary functions.